packages/react/src/auto/AutoForm.ts

```diff
@@ -145,13 +147,18 @@ export const useAutoForm = <
   validateTriggersFromMetadata(metadata);
 
   // filter down the fields to render only what we want to render for this form
+
   const fields = useFormFields(metadata, props);
   const isDeleteAction = metadata && isActionMetadata(metadata) && metadata.action.isDeleteAction;
   const isGlobalAction = action.type === "globalAction";
   const operatesWithRecordId = !!(metadata && isActionMetadata(metadata) && metadata.action.operatesWithRecordIdentity);
   const modelApiIdentifier = action.type == "action" ? action.modelApiIdentifier : undefined;
   const isUpsertMetaAction = metadata && isActionMetadata(metadata) && fields.some((field) => field.metadata.fieldType === FieldType.Id);
   const isUpsertWithFindBy = isUpsertMetaAction && !!findBy;
+  const hasCustomChildren = !!props.children;
+  const fieldPathsToValidate = hasCustomChildren
+    ? [] // With custom children, do not validate fields before sending them to avoid blocking submissions due to missing required fields
+    : fields.map(({ path }) => path);
```

Comment:
Should we wrap this inside a `useMemo` with a `[hasCustomChildren]` as the dep array?

Reply:
Good call. Updated to add a `useMemo` to avoid remapping on fields